### DISCUSSION TITLE
NextJS page flash on mobile swipe back

### DISCUSSION THREAD

**Discussion by randomprogramming** [vercel/next.js#55516] [Help]
### Summary

I recently put my NextJS app into production and I am facing an issue on mobile devices: when I swipe from the left side to go backwards a page, the current page slides to the right along with my finger, and after I let go of the screen, the current page flashes back to its place and then it starts redirecting. How to fix this flickering?

**Comment by joeltran86** [comment]
Hi @randomprogramming 

Have you solved this problem? Do you have any tips to share?
